apps/web/src/routes/GroupWelcomePage/GroupWelcomePage.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import HyloHTML from 'components/HyloHTML'
+import { useViewHeader } from 'contexts/ViewHeaderContext'
+import useRouteParams from 'hooks/useRouteParams'
+import getGroupForSlug from 'store/selectors/getGroupForSlug'
+
+function GroupWelcomePage () {
```

Comment:
Checked this path with a non-member of a group and it throws error 'group is null'. Forward to the public facing about page or not found?